### ISSUE TITLE
Crashes On Ubuntu 2204

### ISSUE THREAD

**Issue by Corgile** [mjakeman/extension-manager#540]
**Describe the bug**
Click Browse, Crash and quit.
See video:

[Screencast from 12-13-2023 08:35:24 PM.webm](https://github.com/mjakeman/extension-manager/assets/48396880/dc591207-352b-41a9-bdfd-503bc6cd516c)

**System Information:**

```shell
❯ uname -a
Linux bryant 6.2.0-37-generic #38~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Nov  2 18:01:13 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

```

**Additional context**

Just open app and click browse

**Comment by oscfdezdz** [comment]
Hi, Ubuntu 22.04 ships with Extension Manager 0.3.0 (seems to be the one used in the screen recording) which is a version that is more than a year old. Have you tried to reproduce the crashes in a newer version?

**Comment by Jai-GAY** [comment]
> Ubuntu 22.04 ships with Extension Manager 0.3.0

May I know how we can get the latest version? we use sudo apt-get install gnome-shell-extension-manager to install it.
What are the steps for Ubuntu Jammy 22.04 64-bit?
 
![extension-mgr_ver](https://github.com/mjakeman/extension-manager/assets/12781023/5b7669e2-25b4-4c15-88ab-08e26726bc45)

**Comment by Jai-GAY** [comment]
> The easiest way is to install it via Flatpak,

Does it mean we can get a newer version than 0.3.0, version > 0.3.0?

**Comment by oscfdezdz** [comment]
> Does it mean we can get a newer version than 0.3.0, version > 0.3.0?

Yes, via Flatpak you can have the latest version available, 0.4.3 and update to newer versions when they are released.
